### ISSUE TITLE
Add support for python 3.10 lambda runtime

### ISSUE THREAD

**Issue by nicornk** [aws-cloudformation/cfn-lint#2700]
### CloudFormation Lint Version

cfn_lint-0.77.1-py3-none-any.whl

### What operating system are you using?

Linux

### Describe the bug

cfn-lint does not support newly released lambda python 3.10 runtime yet:

E3030 You must specify a valid value for Runtime (python3.10). Valid values are ["dotnet6", "dotnetcore1.0", "dotnetcore2.0", "dotnetcore2.1", "dotnetcore3.1", "go1.x", "java11", "java8", "java8.al2", "nodejs", "nodejs10.x", "nodejs12.x", "nodejs14.x", "nodejs16.x", "nodejs18.x", "nodejs4.3", "nodejs4.3-edge", "nodejs6.10", "nodejs8.10", "provided", "provided.al2", "python2.7", "python3.6", "python3.7", "python3.8", "python3.9", "ruby2.5", "ruby2.7"]

### Expected behavior

Should not error out.

### Reproduction template

```yaml
  FunctionLambda:
    Type: AWS::Serverless::Function
    Properties:
      Architectures:
        - 'arm64'
      ...
      Runtime: python3.10
```

**Comment by kddejong** [comment]
Covered by automated PR #2698

**Comment by kddejong** [comment]
Released in v0.77.2
